### ISSUE TITLE
[ONNX] The shape of PReLU weight is wrong

### ISSUE THREAD

**Issue by daquexian** [pytorch/pytorch#21271]
## 🐛 Bug

The shape of PReLU weight is incompatible with ONNX document. 

The shape of weight should be [channels, 1, 1] according to [ONNX document](https://github.com/onnx/onnx/blob/master/docs/Operators.md#PRelu), however, the actual shape is [channels] instead.

It causes the exported ONNX model illegal. Related issue: https://github.com/daquexian/onnx-simplifier/issues/7

## To Reproduce

Code sample:
```python
import torch
from torch import nn
import numpy as np
import onnx
import onnxruntime as rt


shape = (2, 16, 96, 96)

def generate_model():
    net = nn.PReLU(16)
    model_name = 'only_relu.onnx'
    dummy_input = torch.randn(*shape)
    torch.onnx.export(net, dummy_input, model_name, input_names=['input'], output_names=['output'])
    model = onnx.load(model_name)
    return model


def forward(model, inputs):
    sess = rt.InferenceSession(model.SerializeToString())
    outputs = [x.name for x in sess.get_outputs()]
    res = dict(zip(outputs, sess.run(outputs, inputs)))
    return res


forward(generate_model(), {'input': np.random.rand(*shape).astype(np.float32)})
```

Error:

> RuntimeError: Method run failed due to: [ONNXRuntimeError] : 1 : GENERAL ERROR : /onnxruntime_src/onnxruntime/core/providers/cpu/math/element_wise_ops.h:341 void onnxruntime::BroadcastIterator::Init(int64_t, int64_t) axis == 1 || axis == largest was false. Attempting to broadcast an axis by a dimension other than 1. 16 by 96

## Expected behavior

The model runs correctly.

## Environment

PyTorch version: 1.1.0
Is debug build: No
CUDA used to build PyTorch: 9.0.176

OS: Manjaro Linux
GCC version: (GCC) 8.3.0
CMake version: version 3.14.4

Python version: 3.7
Is CUDA available: No

Versions of relevant libraries:
[pip3] numpy==1.16.2
[pip3] torch==1.1.0
[pip3] torchvision==0.2.1
[conda] Could not collect

**Comment by ezyang** [comment]
I can help review a patch that fixes this.

**Comment by daquexian** [comment]
@ezyang Thanks! Could you provide the right way to fix it? The weight [here](https://github.com/pytorch/pytorch/blob/master/torch/onnx/symbolic_opset9.py#L396) has the type `prim::Param`. I haven't found a way to modify it.

**Comment by BowenBao** [comment]
> @ezyang Thanks! Could you please give directions on how to fix it? The weight [here](https://github.com/pytorch/pytorch/blob/master/torch/onnx/symbolic_opset9.py#L396) has the type `prim::Param`. I haven't found a way to modify it.

You can try inserting ```onnx::unsqueeze``` for the weight
```
weight = g.op("Unsqueeze", axes_i=[1, 2])
return g.op("PRelu", self, weight)
```
If the weight is ```prim::Param``` or Constant, this unsqueeze will be optimized away if ```do_constant_folding``` is turned on while exporting.

**Comment by daquexian** [comment]
@ezyang @BowenBao Thanks! I have proposed the PR: #21330

**Comment by daquexian** [comment]
Solved in #21330
